js/app.js

```diff
@@ -24,6 +24,19 @@
 
 	// implement function here
 
+	const newShoppingListItem = (item, price) => {
+		if ( typeof item !== 'string' || typeof price !== 'number') {
+			throw new Error('Parameter type error.');
+		}
+		return {item, price};
+	};
+
+	const listItem1 = newShoppingListItem('milk', 3);
+	const listItem2 = newShoppingListItem('eggs', 1);
+	const listItem3 = newShoppingListItem('bacon', 7);
+	const listItem4 = newShoppingListItem('bagels', 4.95);
+	const listItem5 = newShoppingListItem('filet', 25);
```

Comment:
I presume these are to test your code? I would remove them before submitting, though your idea is sound.